cmd/kubemetrics/main.go

```diff
@@ -124,13 +132,19 @@ func main() {
 	klog.Infof("Using scrape interval of %f", scrapeInterval.Seconds())
 
 	// initialize kubernetes client and cluster cache
-	clusterCache, err := newKubernetesClusterCache()
+	k8sClient, clusterCache, err := newKubernetesClusterCache()
 	if err != nil {
 		panic(err.Error())
 	}
 
+	// Create ConfigFileManager for synchronization of shared configuration
+	confManager := config.NewConfigFileManager(&config.ConfigFileManagerOpts{
+		BucketStoreConfig: env.GetKubecostConfigBucket(),
+		LocalConfigPath:   "/",
```

Comment:
This is me short-cutting a left-over design that I think we _will_ eventually use. To give more context, the Storage implementations take a prefix directory to prepend -- the original idea was to basically do `NewStorage(env.GetConfigPath())` however since there were so many odd discrepancies with the the CONFIG_PATH, I punted. So basically the local config storage will just always default to the root and expect a full path for each config file. This is more of a work-around to introduce a centralized management of configuration and the effort to migrate all existing configurations to this introduced some heavy scope creep. :)

Reply:
What do you think about eliminating `LocalConfigPath` as an exported field and instead just defaulting behavior in the package to `"/"` until the option to set a prefix in the constructor is available?

Reply:
There is an option to set the prefix in the constructor, and it _is_ the default. That's why I'm exposing this field and setting both.

Reply:
Gotcha, sounds good.